docs/index.yml

```diff
@@ -96,3 +96,9 @@ landingContent:
             url: https://officespdev.uservoice.com/
           - text: COM or VSTO add-ins documentation
             url: /office/client-developer/office-client-development
+          - text: Office Add-ins PnP samples repo
+            url: https://github.com/OfficeDev/PnP-OfficeAddins
+          - text: Office Add-ins blogs
+            url: https://developer.microsoft.com/en-us/office/gallery/?filterBy=Add-ins,Blog%20Posts
```

Comment:
Do we have a locale agnostic version of this link (i.e. without en-us)?

Reply:
Yasssssssss ooops! My bad. Fixed both URLS!